src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const splitedStrs = sourceString.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `splitedStrs`, as it does not clearly convey the contents of the variable. Consider using a more descriptive name like `styleDeclarations` or `cssDeclarations` to enhance code readability.